cluster/store/meta_class.go

```diff
@@ -50,7 +50,14 @@ func (m *metaClass) ClassInfo() (ci ClassInfo) {
 	return ci
 }
 
-func (m *metaClass) MultiTenancyConfig() (cfg models.MultiTenancyConfig, version uint64) {
+func (m *metaClass) version() uint64 {
+	if m == nil {
+		return 0
+	}
+	return max(m.ClassVersion, m.ShardVersion)
+}
+
+func (m *metaClass) MultiTenancyConfig() (mc models.MultiTenancyConfig, v uint64) {
 	if m == nil {
 		return
 	}
```

Comment:
**thought (if-minor):** i know it wasn't changed in this PR, but shall the check be after the RWMutex 🤔

Reply:
I believe this is safe since we are not accessing m but only checking its value. The caller might encounter a case where m == nil if the class does not exist in its map